bpfman-operator/controllers/bpfman-agent/fentry-program.go

```diff
@@ -130,54 +153,23 @@ func (r *FentryProgramReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 		return ctrl.Result{Requeue: false}, nil
 	}
 
-	// Get existing ebpf state from bpfman.
-	programMap, err := bpfmanagentinternal.ListBpfmanPrograms(ctx, r.BpfmanClient, internal.Tracing)
-	if err != nil {
-		r.Logger.Error(err, "failed to list loaded bpfman programs")
-		return ctrl.Result{Requeue: true, RequeueAfter: retryDurationAgent}, nil
+	// Create a list of fentry programs to pass into reconcileCommon()
+	var fentryObjects []client.Object = make([]client.Object, len(fentryPrograms.Items))
+	for i := range fentryPrograms.Items {
```

Comment:
I searched for a way to do this that was more direct with no luck :/ this will work

Reply:
I also searched some more after we talked, but same thing here.